src/app/auth/auth.component.ts

```diff
@@ -105,25 +110,31 @@ export class AuthComponent implements OnInit, OnDestroy {
     }
 
     if (!this.authService.isReturnedStateSame(state)) {
+      this.logger.info('Received incorrect state, continue waiting for correct state');
       return;
     }
 
+    this.logger.info('Retrieving access token from Github');
+
     const accessTokenUrl = `${AppConfig.accessTokenUrl}/${oauthCode}/client_id/${AppConfig.clientId}`;
     fetch(accessTokenUrl).then(res => res.json())
       .then(data => {
           if (data.error) {
             throw(new Error(data.error));
           }
           this.authService.storeOAuthAccessToken(data.token);
+          this.logger.info('Sucessfully obtained access token');
         }
       )
       .catch(err => {
+        this.logger.info('Error in data fetched from access token URL');
```

Comment:
Perhaps we can log the `err` here

Reply:
Yup, it will be better if we log the `err`s! For the state, the actual value of the `state` won't really be of importance to us, so I think it is sufficient to log that the `state` checking has failed. Thanks for the suggestions!:)